ev3dev/core.py

```diff
@@ -605,9 +616,20 @@ def time_sp(self):
     @time_sp.setter
     def time_sp(self, value):
         self.set_attr_int('time_sp', value)
+# ~autogen
 
+    def wait_for_running(self):
+        while True:
+            if 'running' in self.state:
+                return
+            sleep(0.01)
```

Comment:
Regarding where to put these wait_for_XYZ() methods, agreed they are not core functionality. I'll create a child Motor class in my common.py and move these there.

As for common.py, any chance I can talk you into letting me move this into the ev3dev directory?  That way someone could easily "import ev3dev.common".  Maybe helper.py would be a more appropriate name.

I think we'll end up with a lot of good stuff in common/helper that will make life much easier for anyone programming EV3 in python.  The easier it is to find/use the better.